Cargo.toml

```diff
@@ -14,20 +14,20 @@ license = "Apache-2.0"
 
 
 [dependencies]
-cxx = "1.0.2"
+cxx = "1.0"
```

Comment:
Since this repo doesn't use `Cargo.lock`, it is better to specify the exact version so that the build is reproducible. Semantic versioning works regardless.

Reply:
thanks for your suggestion. I'm going to refactor my code next week. maybe you can involve in :)